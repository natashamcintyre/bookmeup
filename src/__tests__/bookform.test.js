import React from 'react'
import BookForm from '../components/bookForm'

describe('bookform', () => {
  it('renders without crashing', () => {
    const component = mount(<BookForm />)
    expect(component).toMatchSnapshot()
  })

  it('has submit button', () => {
    const component = mount(<BookForm />)
    expect(component.exists('button#submit')).toBe(true)
  })
})
