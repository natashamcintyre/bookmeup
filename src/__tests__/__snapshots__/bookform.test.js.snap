// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bookform renders without crashing 1`] = `ReactWrapper {}`;
